components/global-header/search.jsx

```diff
@@ -8,18 +8,18 @@
 // ### React
 import React from 'react';
 
-// ### Lookup
-import Lookup from '../lookup';
+// ### Combobox
+import Combobox from '../combobox';
 
 // ## Constants
 import { GLOBAL_HEADER_SEARCH } from '../../utilities/constants';
 
 /**
- * The Global Header Search component is currently a Lookup. In the future this wrapper may provide additional presets or features.
+ * The Global Header Search component is currently just a Combobox. In the future this component will be far more robust and adhere to SLDS standards
  */
 const GlobalHeaderSearch = (props) => (
 	<div className="slds-global-header__item slds-global-header__item_search">
-		<Lookup iconPosition="left" {...props} />
+		<Combobox {...props} />
```

Comment:
there's an issue with onChange when typing in the search. It doesn't need to really work, but it'd be nice to not have an error.
![screen shot 2019-02-16 at 5 05 03 pm](https://user-images.githubusercontent.com/1290832/52906633-17657700-320d-11e9-9dc2-6dc0f0810bb9.png)

Reply:
This was actually a bug with Combobox. handleInputChange was not properly checking for this.props.events && this.props.events.onChange. I've fixed it but also updated the storybook examples to make sure onSelect is passed correctly :)